src/components/SpecialResolution/Memorandum.vue

```diff
@@ -117,8 +118,18 @@
                     </span>
                   </template>
                 </v-checkbox>
+                <span
+                  v-if="!hasResolutionOnFile"
+                  class="black-bold-font"
+                >
+                  Upload a new full set of the memorandum PDF document
+                </span>
+                <v-spacer class="spacer" />
+                <UploadRules
```

Comment:
Probably should rename this component, `UploadRulesOrMemorandum` or make it generic?